ui/imports/shared/controls/chat/LinkPreviewCard.qml

```diff
@@ -0,0 +1,106 @@
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Components 0.1
+
+import shared.status 1.0
+
+CalloutCard {
+    id: root
+    
+    /// The title of the callout card
+    required property string title
+    required property string description
+    required property string footer
+    property StatusAssetSettings logoSettings: StatusAssetSettings {
+        width: 28
+        height: 28
+        bgRadius: bgWidth / 2
+    }
+    
+    property string bannerImageSource: ""
+
+    signal clicked()
+
+    borderWidth: 1
+    padding: borderWidth
+    implicitHeight: 290
+    implicitWidth: 305
+
+    ColumnLayout {
```

Comment:
This better be a `contentItem` with no `anchors.fill` so that the auto-sizing works correctly